cartographer_ros/cartographer_ros/node_main.cc

```diff
@@ -39,13 +39,15 @@ DEFINE_bool(
 DEFINE_string(
     save_state_filename, "",
     "If non-empty, serialize state and write it to disk before shutting down.");
+DEFINE_double(tf_buffer_cache_time_seconds, 1e2,
```

Comment:
So far, *our* system also worked with just `1e1`... however it would be interesting if there are any cases where a longer period is required, as I asked in my other comment.